client/src/game/gameState.d.tsx

```diff
@@ -168,4 +168,38 @@ export type Player = {
     toString(): string
 }
 
+export type Conclusion = "town" | "mafia" | "cult" | "fiends" | "politician" | "draw";
+
+export type WinCondition = {
+    type: "gameConclusionReached"
+    winIfAny: Conclusion[]
+} | {
+    type: "roleStateWon"
+}
 
+export function translateWinCondition(winCondition: WinCondition): string {
```

Comment:
I know you know why i dont like this

What about in the future when i add litterally 1 more win condition
There should certainly be some sort of loop here that goes through all conclusions in the win if any

Reply:
Ok, sure

Reply:
I want to make sure i was clear

I may add a role that wins if either cult or syndicate wins but not fiends. 

Just make sure its future proof, since we have very few developers.

Reply:
Got it, sounds good